confgenerator/logging_processors.go

```diff
@@ -123,6 +123,44 @@ func (r MultilineRule) AsString() string {
 	return fmt.Sprintf(`"%s"    "%s"    "%s"`, r.StateName, escapedRegex, r.NextState)
 }
 
+type LoggingProcessorParseMultiline struct {
+	Field string
+	Rules []MultilineRule
+}
+
+func (p LoggingProcessorParseMultiline) Components(tag, uid string) []fluentbit.Component {
```

Comment:
As @sophieyfang mentioned in https://github.com/GoogleCloudPlatform/ops-agent/pull/219#issuecomment-1015939175, there is an issue with using the `multiline` plugin as a standalone filter plugin.

We probably should change the underlying generated fluent bit configuration from a standalone multiline filter plugin to an embedded section of a tail input plugin at some point. (Not necessarily a blocker for this PR).

Reply:
I went ahead and took a stab at getting it to work as described here: https://docs.fluentbit.io/manual/pipeline/inputs/tail#multiline-core-v1.8